docs/tidb_http_api.md

```diff
@@ -0,0 +1,52 @@
+#  TiDB HTTP API
+
+`TiDBIP` is the ip of tidb, `10080` is the default status port, and it can be changed in the tidb.toml when start. 
```

Comment:
`TiDBIP` is the ip of the TiDB server. `10080` is the default status port, and it can be changed in tidb.toml when started.